python/servo/devenv_commands.py

```diff
@@ -34,6 +34,17 @@ def cargo(self, params):
         if not params:
             params = []
 
+        cargo_paths = [path.join('components', 'servo'),
+                       path.join('ports', 'cef'),
+                       path.join('ports', 'geckolib')]
+
+        if params and params[0] == "fetch":
+            for cargo_path in cargo_paths:
```

Comment:
I think it makes more sense to inline this at the call site, since this doesn't really benefit from being inside the `cargo` function.